diamm/helpers/object_pagination.py

```diff
@@ -0,0 +1,32 @@
+from collections import OrderedDict
+import re
+
+from rest_framework.response import Response
+from rest_framework.pagination import PageNumberPagination
+
+class ObjectPagination(PageNumberPagination):
+    def replace_page_query(self, url, page_num):
+        regex = "(?P<prefix>&|\?)page=[0-9]+"
+        if re.search(regex, url):
+            return re.sub(regex, "\g<prefix>page=%d" % page_num, url)
+        else:
+            prefix = "&" if url.count("?") else "?"
+            return ("%s%spage=%d" % (url, prefix, page_num))
```

Comment:
We generally use the `.format` method for string formatting, instead of the `printf`-style